packages/paste-core/components/disclosure/__tests__/disclosure.test.tsx

```diff
@@ -161,15 +159,19 @@ describe('Disclosure', () => {
       expect(renderedDisclosureButton.getAttribute('tabindex')).toEqual('0');
       expect(screen.getByTestId('disclosure').id).toEqual('disclosure');
     });
-    it('should render a disclosure open', () => {
-      render(<MockDisclosure visible />);
+    it('should render a disclosure open', async () => {
+      await waitFor(() => {
+        render(<MockDisclosure visible />);
+      });
```

Comment:
This also seemed to be tripping up jest with an incomplete async test. Setting visible causes a state change which wasn't wrapped in `act`. Waiting for it, fixes that, and the potential for the `createEvent` bug to appear.